tests/integration_tests/functional/test_drives.py

```diff
@@ -172,26 +168,20 @@ def test_rescan_dev(test_microvm_with_api, network_config):
     )
 
     losetup = ['losetup', '--find', '--show', fs2.path]
-    loopback_device = None
-    result = None
-    try:
-        result = utils.run_cmd(losetup)
-        loopback_device = result.stdout.rstrip()
-    except ChildProcessError:
-        pytest.skip('failed to create a lookback device: ' +
-                    f'stdout={result.stdout}, stderr={result.stderr}')
-
-    try:
-        response = test_microvm.drive.patch(
-            drive_id='scratch',
-            path_on_host=test_microvm.create_jailed_resource(loopback_device),
-        )
-        assert session.is_status_no_content(response.status_code)
-
-        _check_block_size(ssh_connection, '/dev/vdb', fs2.size())
-    finally:
-        if loopback_device:
-            utils.run_cmd(['losetup', '--detach', loopback_device])
+    rc, stdout, _ = utils.run_cmd(losetup)
+    assert rc == 0
+    loopback_device = stdout.rstrip()
+
+    response = test_microvm.drive.patch(
+        drive_id='scratch',
+        path_on_host=test_microvm.create_jailed_resource(loopback_device),
+    )
+    assert session.is_status_no_content(response.status_code)
+
+    _check_block_size(ssh_connection, '/dev/vdb', fs2.size())
+
+    rc, _, _ = utils.run_cmd(['losetup', '--detach', loopback_device])
```

Comment:
The initial behavior is changed. In the initial version if   `assert session.is_status_no_content(response.status_code)` failed, the loopback device got removed. In this version of the code, it does not, right? Do we want that?